### DISCUSSION TITLE
Issue with Selecting a Classifier

### DISCUSSION THREAD

**Discussion by tonygem2022** [iTwin/itwinjs-core#5758] [Q&A]
We have a reality model where we consistently find it near impossible to select an element.

Easiest to explain this in the attached video.

Effectively you see me moving my cursor across the entire pipe and it is only until i reach the end of the pipe that i am able to select the element. This is not too intuitive even so and is often hit and miss.

If i turn on the classifier, you see it more than adequately covers the full diameter of the pipe and when i click on it then turn it off, the pipe is selected as intended, BUT, we want to be able to select the pipe (or any other element) any where we choose to along the pipe itself but for some reason, we re unable to do this. UX is low accordingly and becomes frustrating.

Timely guidance welcomed please.

Thanks. 

Tony

https://github.com/iTwin/itwinjs-core/assets/108276006/94ce3aa8-b135-41be-a6d2-a77eb85dc40c

**Comment by tonygem2022** [comment]
Hoping i can receive a support response please?

Thanks.

Tony

**Reply by pmconne** [reply]
@tonygem2022, it's going to be very tricky to diagnose this via GitHub discussions. We tried to get in direct contact with you last week. Please send an email to paul dot connelly at bentley dot com.
